src/helpers/utils.ts

```diff
@@ -62,3 +60,35 @@ export function requestContainsCredentialType(
     }
   });
 }
+
+export function toChecksumAddress(
+  address: string,
+  chainId: string | null = null,
+) {
+  const keccak = require("keccak");
```

Comment:
This is odd. It should be an import.

Also we should use the `import { keccak_256 } from '@noble/hashes/sha3';` from the `@noble/hashes` library. It looks like viem and others use it under the hood and it is audited.

https://github.com/wevm/viem/blob/007b9eef021357fb340cd230be2c0294237fcb91/src/utils/address/getAddress.ts#L20

I think the only real change is that 

`const keccakHash = keccak("keccak256").update(prefix + stripAddress).digest("hex");` -> `keccak256(prefix + stripAddress)`